runtime/src/lib.rs

```diff
@@ -368,6 +380,7 @@ impl pallet_rmrk_core::Config for Runtime {
 	type MaxResourcesOnMint = MaxResourcesOnMint;
 	type NestingBudget = NestingBudget;
 	type WeightInfo = pallet_rmrk_core::weights::SubstrateWeight<Runtime>;
+	type ProducerOrigin = frame_system::EnsureSignedBy<AllowedMinters, Self::AccountId>;
```

Comment:
This `ProducerOrigin` type will be used inside the mint and collection extrinsics, right?

Reply:
Correct.

Reply:
Nice! 🚀